src/psyclone/tests/domain/lfric/lfric_loop_test.py

```diff
@@ -333,21 +364,25 @@ def test_loop_stop_expr(dist_mem):
     loops = sched.walk(DynLoop)
     ubound = loops[1].stop_expr
     assert isinstance(ubound, ArrayReference)
-    assert ubound.symbol.name == "last_cell_all_colours"
     assert ubound.indices[0].name == "colour"
     if dist_mem:
+        assert ubound.symbol.name == "last_halo_cell_all_colours"
         assert isinstance(ubound.indices[1], Literal)
         assert ubound.indices[1].value == "1"
         # Alter the loop so that it goes to the full halo depth
         loops[1]._upper_bound_halo_depth = None
         ubound = loops[1].stop_expr
         assert isinstance(ubound.indices[1], Reference)
         assert ubound.indices[1].symbol.name == "max_halo_depth_mesh"
+    else:
+        assert ubound.symbol.name == "last_edge_cell_all_colours"
 
 
 def test_loop_stop_expr_intergrid(dist_mem):
     ''' Test the stop_expr property for a loop containing an
-    inter-grid kernel. '''
+    inter-grid kernel.
```

Comment:
I would suggest single quotes around `'stop_expr'` in the line above.

Reply:
done